README.ar-DZ.md

```diff
@@ -0,0 +1,135 @@
+
+![](https://user-images.githubusercontent.com/8186664/44953195-581e3d80-aec4-11e8-8dcb-54b9db38ec11.png)
+
+</div>
+
+- معاينة: http://preview.pro.ant.design
+- الصفحة الرئيسية: http://pro.ant.design
+- توثيق: http://pro.ant.design/docs/getting-started
+- سجل التغيير: http://pro.ant.design/docs/changelog
+- الأسئلة الشائعة: http://pro.ant.design/docs/faq
+- مرآة الموقع بالصينية: http://ant-design-pro.gitee.io
+
+## 4.0 صدر الآن! 🎉🎉🎉
+
+[الإعلان عن Ant Design Pro 4.0.0](https://medium.com/ant-design/ant-design-pro-v4-is-here-6f23098ae9d9)
+
+## توظيف الترجمة :loudspeaker:
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=5996021)